docs/CHANGELOG.asciidoc

```diff
@@ -1,20 +1,20 @@
 // Use these for links to issue and pulls. Note issues and pulls redirect one to
 // each other on Github, so don't worry too much on using the right prefix.
-// :issue: https://github.com/elastic/elasticsearch/issues/
-// :pull: https://github.com/elastic/elasticsearch/pull/
+:issue: https://github.com/elastic/elasticsearch/issues/
+:pull: https://github.com/elastic/elasticsearch/pull/
 
 = Elasticsearch Release Notes
 
 == Elasticsearch 7.0.0
 
 === Breaking Changes
 
-<<write-thread-pool-fallback, Removed `thread_pool.bulk.*` settings and
-`es.thread_pool.write.use_bulk_as_display_name` system property>> ({pull}29609[#29609])
+write-thread-pool-fallback, Removed `thread_pool.bulk.*` settings and
```

Comment:
Is this supposed to be a link to https://www.elastic.co/guide/en/elasticsearch/reference/master/breaking_70_settings_changes.html#write-thread-pool-fallback ?